dts/arm/silabs/siwg917.dtsi

```diff
@@ -183,6 +183,10 @@
 
 			#address-cells = <1>;
 			#size-cells = <0>;
+			silabs,port-count = <4>;
+			silabs,max-pin-usage-count = <8>;
```

Comment:
Code should use the pin mask constructed from `ngpios` and `gpio-reserved-ranges`, which is already available as the `.common.port_pin_mask` member of the config struct on each port. No need for another dts property.